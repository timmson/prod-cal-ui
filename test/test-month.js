import React from "react"
import renderer from "react-test-renderer"
import Month from "../src/month"

describe("Month should", () => {

	test("be equal to snapshot", () => {
		const m = {
			name: "October",
			number: "10",
			working: {
				days: 10,
				hours: 100
			},
			weeks: []
		}

		const component = renderer.create(<Month year={2020} month={m} isSelected={false}/>)

		expect(component.toJSON()).toMatchSnapshot()
		component.unmount()
	})

})