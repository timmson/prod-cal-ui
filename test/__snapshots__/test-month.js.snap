// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Month should be equal to snapshot 1`] = `
<div
  className="col float-right mt-4"
>
  <table
    className="calendar "
    onClick={[Function]}
  >
    <thead>
      <tr>
        <td
          className="month"
          colSpan="8"
        >
          October
          '
          20
           - 
          <span>
            [working 
            10
            d /
            100
            h]
          </span>
        </td>
      </tr>
    </thead>
    <tbody>
      <tr
        className="dayOfWeek"
      >
        <td
          className="work"
        >
          #w
        </td>
        <td
          className="work"
        >
          mo
        </td>
        <td
          className="work"
        >
          tu
        </td>
        <td
          className="work"
        >
          we
        </td>
        <td
          className="work"
        >
          th
        </td>
        <td
          className="work"
        >
          fr
        </td>
        <td
          className="holiday"
        >
          sa
        </td>
        <td
          className="holiday"
        >
          su
        </td>
      </tr>
    </tbody>
  </table>
</div>
`;
